components/base/Typography.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import classnames from "classnames";
+import { POPPINS, ROBOTO } from "../typography";
+import {
+  HEADLINE_TEXT,
+  PARAGRAPH_TEXT,
+  WHITE_PRIMARY,
+  GREY_5,
+  GREY_4,
+} from "../colors";
+
+// Typography function
+function Typography({ text, styleVariant, tag, isDark }: TypographyProps) {
+  return (
+    <TypographyStyles
+      as={`${tag}`}
```

Comment:
> as={\`${tag}\`}

@sausman, this is what I'd come up with for dynamic tags in styled-components, but it seems to be failing checks. Would you be able to take a look at this when you get a chance?

Reply:
Hmm yeah that's bizarre, something with TypeScript. I tried updating dependencies and that didn't fix it.